portal-frontend/src/app/features/public/application/submission/subd-details/subd-details.component.scss

```diff
@@ -1,9 +1,17 @@
-@use '../../../../../../styles/functions' as *;
+@use '../../../../../../styles/functions.scss' as *;
```

Comment:
Not sure if you need `.scss`, imports in other files don't have it, if it's not needed, we should remove it to keep it consistent across all files